.changes/unreleased/tinymce-TINY-11763-2025-02-19.yaml

```diff
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: Deleting an empty block within <li> would move cursor at the end of the <li>.
```

Comment:
That is one long message. I wonder if you could split it up into two log items with the same issue ID since its fixing two bugs.